fixes/github-issue-create.js

```diff
@@ -169,53 +225,99 @@ async function getTopCommittersOfRepository(targetOrg, targetRepository) {
 }
 
 /**
- * Create an issue on Github with labels and all on the target repository.
+ * Create an issue on GitHub with labels and all on the target repository.
  *
  * @param {object} options The rule configuration.
- * @returns {object} Returns issue after adding it via the Github API.
+ * @param {number} assigneeSelectCount counter for which assignee was selected
+ * @param {array<object>} assignees array of contributors
+ * @returns {object} Returns issue after adding it via the GitHub API.
  */
-async function createIssueOnGithub(options) {
-  try {
-    return await this.Octokit.request('POST /repos/{owner}/{repo}/issues', {
-      owner: targetOrg,
-      repo: targetRepository,
-      title: options.issueTitle,
-      body: InternalHelpers.generateIssueBody(options),
-      labels: options.issueLabels,
-      assignees: issuesAssignees
-    })
-  } catch (e) {
-    console.error(e)
-  }
+async function createIssueOnGithub(options, assigneeSelectCount, assignees) {
+  // This might not be needed
+  let tempAssigneeSelectCount = assigneeSelectCount
+  return await this.Octokit.request('POST /repos/{owner}/{repo}/issues', {
+    owner: targetOrg,
+    repo: targetRepository,
+    title: options.issueTitle,
+    body: InternalHelpers.generateIssueBody(options),
+    labels: options.issueLabels,
+    assignees: issuesAssignees
+  }).catch(error => {
+    if (
+      error.status === 422 &&
+      error.message.indexOf('Validation Failed: ') !== -1 &&
+      error.message.indexOf('"field":"assignees","code":"invalid"}')
+    ) {
+      issuesAssignees = []
+      if (
+        tempAssigneeSelectCount <= assignees.data.length &&
+        tempAssigneeSelectCount <= 6
+      ) {
+        tempAssigneeSelectCount++
+        issuesAssignees.push(assignees.data[tempAssigneeSelectCount].login)
```

Comment:
Does this actually work? Assume the initial invocation of this function is called with an 8 element array `assignees`.
The first recursive call we be done with a single element array with `assignee[1]` from the initial call. After that it will fail? Will create a test for that scenario.

Reply:
Yes, this works, I tested this extensively. The recursion will stop when either of the following conditions is true:
- assigneeSelectCount is **bigger** or **equal** the maximum length of the assignees array retrieved from GitHub
- assigneeSelectCount reaches or exceeds (somehow) the maximum assignee retry counter (which is a constant and currently set to 5)

If any of the above statements are true, it will create an issue without having anyone assigned.